util/restmapper/cached.go

```diff
@@ -58,7 +58,7 @@ func (c *cached) flush() error {
 }
 
 func (c *cached) shouldFlushOn(err error) bool {
-	switch err.(type) {
+	switch err.(type) { // nolint:gocritic
```

Comment:
Is there a way we can resolve this rather than ignore it?

Reply:
thanks, updated! now checked like bellow 
```
if _, ok := err.(*meta.NoKindMatchError); 
```